docs/v3_services.rst

```diff
@@ -1,89 +1,103 @@
 SecureDrop V3 Onion Services
 ============================
-Tor onion services provide anonymous inbound connections to websites and other
```

Comment:
It's worth keeping the "what is an onion service and what does it do?" line IMO, just for admins and other folk that don't spend all their time on the DarkWeb, and may not immediately twig why this is important.

Reply:
IMO our docs have too much such preamble and my bias would be to keep them to the requirements ("enable v3 today! here's how") instead of taking as long as we do to explain why; however, maybe this is more of a long-term structural question about our documentation and should be discussed separately. Added this back in 71d4739.

Reply:
tiny typo slipped in: TOr->Tor

Reply:
Oops- fixed in 61fa02a, thanks for catch